devel/addon/certmanager/install.sh

```diff
@@ -28,7 +28,7 @@ FEATURE_GATES="${FEATURE_GATES:-ExperimentalCertificateSigningRequestControllers
 # As Feature Gates are added/removed, these lists should be updated.
 declare -a FEATURE_GATES_CONTROLLER_ALL=(\
 "AllAlpha","AllBeta","ValidateCAA","ExperimentalCertificateSigningRequestControllers",\
-"ExperimentalGatewayAPISupport","AdditionalCertificateOutputFormats")
+"ExperimentalGatewayAPISupport","AdditionalCertificateOutputFormats","ServerSideApply")
```

Comment:
Note: it might be good if this script logged something like 'Installing cert-manager with feature gates X,Y,Z enabled'

I've forgotten that it works that way a few times when I am running it locally to test some changes

Reply:
Good idea, will add that in.

Reply:
I've added a log output, PTAL